### ISSUE TITLE
Not achieving same accuracy of target model

### ISSUE THREAD

**Issue by Tidistamatiou** [AdrienBenamira/membership_inference_attack#1]
Hi, 

Thank you for supplying code for this Membership Inference Attack, I've stumbled upon an issue I'm having when trying to recreate the results shown in the paper for the CIFAR10 dataset. When running the main.py script with a copy of the config file found in the results section folder called 'overfitting_CIFAR10_overfitting_2019_05_11_09_48_18' and adding a small print statement to report the best accuracy found after training the (target) model for the specified amount of epochs, my accuracy is much lower than the accuracy reported. The accuracy of the target model, gained by printing the 'best_acc' variable from the train_model function in trainer.py, after training for 100 epochs is only 32.7%. If I train the used model (Net_cifar10) myself with the same parameters I only achieve around 60% accuracy after 100 epoch.

Could you maybe test this yourself or explain why my performance might differ from the reported performance?

**Comment by FakeEyes2Wo** [comment]
+1
